packages/client/src/StreamrClient.ts

```diff
@@ -554,14 +555,27 @@ export class StreamrClient {
         return this.node.getNode()
     }
 
+    getEntryPoints(): PeerDescriptor[] {
+        return this.node.getEntryPoints()
+    }
+
     async setProxies(
         streamDefinition: StreamDefinition,
-        nodeIds: string[],
+        nodeDescriptors: JsonPeerDescriptor[],
         direction: ProxyDirection,
         connectionCount?: number
     ): Promise<void> {
         const streamPartId = await this.streamIdBuilder.toStreamPartID(streamDefinition)
-        await this.node.setProxies(streamPartId, nodeIds, direction, connectionCount)
+        await this.node.setProxies(streamPartId, nodeDescriptors, direction, connectionCount)
+    }
+
+    /**
+     * Used to set known entry points for a stream. If known are not set they 
+     * will be automatically discovered from the Streamr Network.
+    */
+    async setStreamEntryPoints(streamDefinition: StreamDefinition, nodeDescriptors: JsonPeerDescriptor[]): Promise<void> {
```

Comment:
Do we set entry points per stream or per stream part? From line 578 it seems that we handle entry points per stream part, but the parameter name implies otherwise. 

When a user calls this method, is it very likely that his intention is to set the entry point for the whole stream? If that's the case, we could maybe query the partition count of the stream and set the entry points for all partitions. (Ether here or in network node level.)

Reply:
Per stream partition, renamed